CRM/Civihoneypot/Form/HoneypotSettings.php

```diff
@@ -32,14 +32,15 @@ function processHoneypotOptions($mode) {
 		'api' => array('label_field' => 'title'),
 	  ));
       $this->add('text', 'field_names', ts('Field Names', array('domain' => 'com.elisseck.civihoneypot')));
+      $this->add('checkbox', 'protect_all', ts('Protect All Pages', array('domain' => 'com.elisseck.civihoneypot')));
 	  $this->add('text', 'limit', ts('Time Limiter', array('domain' => 'com.elisseck.civihoneypot')));
 	  $this->add('textarea', 'ipban', ts('Banned IP Addresses', array('domain' => 'com.elisseck.civihoneypot')));
-      $this->addRule('form_ids', 'Enter Form IDs', 'required');
```

Comment:
@mrlavalava @elisseck I feel like we possibly want to keep this but move it into a validateForm hook so that if they turn it on but don't check the protect all box and they haven't entered in any form_ids we invalidate the form